client/src/com/mirth/connect/client/ui/Frame.java

```diff
@@ -1,5133 +1,5106 @@
+        channelEditPanel.editChannel(channel);
+    }
+
+    /**
+     * Changes the current content page to the Alert Editor with the new alert specified as the
+     * loaded one.
+     */
+    public void setupAlert(Map<String, Map<String, String>> protocolOptions) {
+        setBold(viewPane, UIConstants.ERROR_CONSTANT);
+        setCurrentContentPage(alertEditPanel);
+        setFocus(alertEditTasks);
+        setVisibleTasks(alertEditTasks, alertEditPopupMenu, 0, 0, false);
+        alertEditPanel.addAlert(protocolOptions);
+    }
+
+    /**
+     * Edits an alert at a specified index, setting that alert as the current alert in the editor.
+     */
+    public void editAlert(AlertModel alertModel, Map<String, Map<String, String>> protocolOptions) {
+        if (alertEditPanel.editAlert(alertModel, protocolOptions)) {
+            setBold(viewPane, UIConstants.ERROR_CONSTANT);
+            setCurrentContentPage(alertEditPanel);
+            setFocus(alertEditTasks);
+            setVisibleTasks(alertEditTasks, alertEditPopupMenu, 0, 0, false);
+        }
+    }
+
+    /**
+     * Edit global scripts
+     */
+    public void editGlobalScripts() {
+        setBold(viewPane, UIConstants.ERROR_CONSTANT);
+        setCurrentContentPage(globalScriptsPanel);
+        setFocus(globalScriptsTasks);
+        setVisibleTasks(globalScriptsTasks, globalScriptsPopupMenu, 0, 0, false);
+        setPanelName("Global Scripts");
+    }
+
+    /**
+     * Sets the current content page to the passed in page.
+     */
+    public void setCurrentContentPage(Component contentPageObject) {
+        if (contentPageObject == currentContentPage) {
+            return;
+        }
+
+        if (currentContentPage != null) {
+            contentPane.getViewport().remove(currentContentPage);
+        }
+
+        contentPane.getViewport().add(contentPageObject);
+        currentContentPage = contentPageObject;
+
+        // Always cancel the current job if it is still running.
+        if (statusUpdaterJob != null && !statusUpdaterJob.isDone()) {
+            statusUpdaterJob.cancel(true);
+        }
+
+        // Start a new status updater job if the current content page is the dashboard
+        if (currentContentPage == dashboardPanel || currentContentPage == alertPanel) {
+            statusUpdaterJob = statusUpdaterExecutor.submit(new StatusUpdater());
+        }
+    }
+
+    /**
+     * Sets the current task pane container
+     */
+    private void setCurrentTaskPaneContainer(JXTaskPaneContainer container) {
+        if (container == currentTaskPaneContainer) {
+            return;
+        }
+
+        if (currentTaskPaneContainer != null) {
+            taskPane.getViewport().remove(currentTaskPaneContainer);
+        }
+
+        taskPane.getViewport().add(container);
+        currentTaskPaneContainer = container;
+    }
+
+    /**
+     * Makes all of the task panes and shows the dashboard panel.
+     */
+    private void makePaneContain
```

Comment:
@pacmano1 noticed that removing this causes issues when the notifications actually run, as it tries to update the menu entry with a count and overwrites the name of the View User API task instead.